Target-MVD-Mathias/Models/Match.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  Match.swift
+//  Target-MVD-Mathias
+//
+//  Created by Mathias Cabano on 9/26/17.
+//  Copyright © 2017 TopTier labs. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+class Match: NSObject, NSCoding {
+  
+  var id: Int
+  var topic: Topic
+  var user: User
+  var title: String
+  var channelId: String
+  var unread: Int
+  var lastMessages: [String]
+  var active: Bool
+  
+  init(id: Int, topic: Topic, user: User, title: String = "", channelId: String = "", unread: Int, lastMessages: [String], active: Bool) {
+    self.id = id
+    self.topic = topic
+    self.user = user
+    self.title = title
+    self.channelId = channelId
+    self.unread = unread
+    self.lastMessages = lastMessages
+    self.active = active
+  }
+  
+  required init?(coder aDecoder: NSCoder) {
```

Comment:
This should be removed if it's not being used, nor is it going to be used in the near future.